mysql-test/suite/binlog_gtid/t/binlog_gtid_log_status_mismatch.test

```diff
@@ -0,0 +1,37 @@
+--source include/have_debug.inc
```

Comment:
This test seems to pass even without the fix. Can you check this?

Reply:
done